lib/ClangImporter/ImportType.cpp

```diff
@@ -2431,6 +2431,19 @@ DefaultArgumentKind ClangImporter::Implementation::inferDefaultArgument(
           return DefaultArgumentKind::EmptyArray;
       }
     }
+  } else if (const clang::TypedefType *typedefType =
+                 type->getAs<clang::TypedefType>()) {
+    // Get the AvailabilityAttr that would be set from CF/NS_OPTIONS
+    if (importer::isUnavailableInSwift(typedefType->getDecl(), nullptr, true)) {
```

Comment:
You can actually use `if (isUnavailableInSwift(typdefType->getDecl()))` here because you're in the `Implementation` (in my patch I had to use this because of layering).

Reply:
I see this is a static method, so never mind.